CONTRIBUTING.md

```diff
@@ -222,6 +222,10 @@ in [`native-tests.json`](.github/native-tests.json) to have its integration test
 
 If you have not done so on this machine, you need to:
 
+* Make sure you have a case-sensitive filesystem. Java development on a case-insensitive filesystem can cause headaches.
+    * Linux: You're good to go.
+    * macOS: Use the `Disk Utility.app` to check. It also allows you to create a case-sensitive volume to store your code projects. See this [blog entry](https://karnsonline.com/case-sensitive-apfs/) for more.
```

Comment:
```suggestion
    * macOS: Use the `Disk Utility.app` to check. It also allows you to create a case-sensitive volume to store your code projects. See this [blog entry](https://karnsonline.com/case-sensitive-apfs/) for more details and if you don't want to ref-ormat and re-install macOS.
```

Reply:
Reinstalling doesn't solve the problem, since you likely do not want to make your entire system be case sensitive: there would be impacts on other applications, most of which probably assume case-insensitivity. I think that adding a volume is the best advice.

Reply:
Ok nevermind then 👍